hosting/src/Home/HomeBanners.ts

```diff
@@ -42,7 +42,7 @@ export default class Home extends Vue {
   currentSlideIndex = 0
   nextSlideInterval?: NodeJS.Timeout = undefined
 
-  get banners(): Array<Omit<Banner, 'slug' | 'dateExpire'>> { return this.$store.state.banners }
+  get banners(): Array<Omit<Banner, 'slug' | 'dateExpire'>> { return this.$store.state.banners.sort((b1, b2) => (b2.priority - b1.priority) || (b1.dateExpire.seconds - b2.dateExpire.seconds)) }
```

Comment:
Please remove `| 'dateExpire'` from the type

Reply:
I deleted | 'dateExpire'.